disco_aws_automation/asiaq_cli.py

```diff
@@ -103,7 +105,91 @@ def _update_s3_configs(self, sandbox_name):
             s3_bucket.upload_file(local_config_path, remote_config_path)
 
 
+class DynamoDbBackupCommand(CliCommand):
+    """
+    CliCommand implementation for managing DynamoDB backups and backup pipelines.
+    """
+    DESCRIPTION = "Manage dynamodb backups and the pipelines that create them."
+
+    @classmethod
+    def init_args(cls, parser):
+        subsub = parser.add_subparsers(title="data pipeline commands", dest="dp_command")
+        subsub.add_parser("init", help="Set up bucket for backup and log data.")
+        backup_parser = subsub.add_parser("backup",
+                                          help="Configure backup for a dynamodb table")
+        backup_parser.add_argument("table_name")
+        restore_parser = subsub.add_parser("restore", help="Restore a dynamodb table from backup")
+        restore_parser.add_argument("table_name")
+        restore_parser.add_argument("--from", dest="backup_dir",
+                                    help="Previous backup to restore from (default: latest)")
+        list_parser = subsub.add_parser("list", help="List existing backups")
```

Comment:
`awscli s3 rm`... 😕 

I'm punting on the S3 cleanup right now, in that it's easy to do manually and not actually closely related to the backup/restore work itself.  Nupur suggested setting up lifecycle events to auto-delete things, and that seems sensible, I just didn't want to let the scope keep creeping.